packages/core/src/components/FacetSearch/FacetSearch.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { Meta, StoryObj } from "@storybook/react";
+import FacetSearchRaw from "./FacetSearch?raw";
+import { FacetSearch } from "./FacetSearch";
+
+const meta: Meta = {
+  title: "Widgets/Facet Search",
+};
+
+export default meta;
+
+export const Sample: StoryObj = {
+  parameters: {
+    docs: {
+      source: { code: FacetSearchRaw },
+      description: {
+        story:
+          "A Sample of an example implementation of the Facet Search design patterned.",
```

Comment:
This description is not showing on storybook. I think it's because this sample is the main story and for some reason storybook doesn't show the description 😅 Maybe using an `mdx` file like we have for some stories would be better.

Also, we have this "Nothing found" label since we don't have an args table:

<img width="1036" alt="Screenshot 2023-10-25 at 16 18 43" src="https://github.com/lumada-design/hv-uikit-react/assets/43220251/bbdebc00-87e8-42a4-b467-f14a259ce21d">

We can remove this with a decorator (example in the file `docs/templates/templateDecorator`).